laser_encoders/README.md

```diff
@@ -27,27 +27,33 @@ You can install laser_encoders using pip:
 
 Here's a simple example of how you can download and initialise the tokenizer and encoder with just one step.
 
-**Note:** By default, the models will be downloaded to the `~/.cache/laser_encoders` directory. To specify a different download location, you can provide the argument `model_dir=path/to/model/directory` to the initialize_tokenizer and initialize_encoder functions
+**Note:** By default, the models will be downloaded to the `~/.cache/laser_encoders` directory. To specify a different download location, you can provide the argument `model_dir=path/to/model/directory` to the initialize_tokenizer and LaserEncoderPipeline functions
```

Comment:
I had a different structure in mind when proposing this change: 
- initialize_encoder - initializes an encoder only; it has the argument `tokenize` set to `False` by default (or maybe even doesn't have this option at all, to create less confusion). 
- initialize_tokenizer - initializes a tokenizer only
- LaserEncoderPipeline - initializes a tokenizer and an encoder and puts them together; it doesn't have the argument `tokenize` because it always includes an encoder and a tokenizer

Reply:
Alright, I'll make the necessary changes.